src/idl_gen_ts.cpp

```diff
@@ -102,6 +83,17 @@ namespace ts {
 // and tables) and output them to a single file.
 class TsGenerator : public BaseGenerator {
  public:
+  struct ImportDefinition {
```

Comment:
not sure why these are now public. I see the `import_set` uses it below, but no one outside this class knows about `ImportDefinition`. It seems like the visibility is incorrect.